tests/FsUnit.MbUnit.Test/beNullTests.fs

```diff
@@ -11,12 +11,29 @@ type ``be Null tests`` ()=
 
     [<Test>] member test.
      ``null should fail to not be Null`` ()=
-        null |> should be Null
+        shouldFail (fun () -> null |> should not' (be Null))
```

Comment:
first 2 test of the test-class have exact same body, but different naming:
- null should be Null
- null should fail to not be Null

So they really test same thing.
There is the same situation in ```be Null tests``` for XUnit and MsTest.
If you take a loot at NUnit tests you'll see that first 2 tests are testing what is stated in their names:
```fsharp
[<Test>] member test.
     ``null should be Null`` ()=
        null |> should be Null

    [<Test>] member test.
     ``null should fail to not be Null`` ()=
        shouldFail (fun () -> null |> should not' (be Null))
```
So I guess there should be no backward-compatibility issues and now test body corresponds to the test name, just like similar test for NUnit